### PR TITLE
Added ryannkim327.json

### PR DESCRIPTION
**Pull request by RyannKim327**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/ryannkim327.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "description": "Ryann Kim Sesgundo, a student from Dalubhasaan ng Lungsod ng Lucena",
+    "repo": "https://github.com/RyannKim327/ryannkim327.github.io",
+    "owner": {
+        "username": "RyannKim327",
+        "email": "weryses19@gmail.com",
+        "twitter": ""
+    },
+    "record": {
+        "CNAME": "ryannkim327.github.io/"
```
Comment (VaibhavSys) [inline domains/ryannkim327.json:10]:
```suggestion
        "CNAME": "ryannkim327.github.io"
```
Invalid CNAME

File: domains/ryannkim327.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "description": "Ryann Kim Sesgundo, a student from Dalubhasaan ng Lungsod ng Lucena",
+    "repo": "https://github.com/RyannKim327/ryannkim327.github.io",
+    "owner": {
+        "username": "RyannKim327",
+        "email": "weryses19@gmail.com",
+        "twitter": ""
```
Comment (VaibhavSys) [inline domains/ryannkim327.json:7]:
```suggestion
```
Remove it if you are going to leave it empty

**Review by VaibhavSys — CHANGES_REQUESTED** [review]

File: domains/ryannkim327.json (hunk 2 above)
Comment (RyannKim327) [reply domains/ryannkim327.json]: i thought it is required, but I commit the suggestion so that it was already removed, thanks btw.

**Review by Stef-DP — APPROVED** [review]
seems good now

**Review by VaibhavSys — APPROVED** [review]

**Pull request merged**